ProgrammingBasics/Arrays/BuildArray/Application.gcomp/Build Array.gviweb

```diff
@@ -97,7 +97,9 @@
 			<Text BaseName="[string]Text" Foreground="[SMColor]#ff044123" Height="[float]22" Id="6d46358c0fcc4d76a8066cc5dbdab265" Left="[float]36" Text="[string]Overview" TextWrapping="[TextWrapping]Wrap" Top="[float]148" Width="[float]81" xmlns="http://www.ni.com/PlatformFramework">
 				<FontSetting FontFamily="Segoe UI" FontSize="12" FontStyle="Bold" Id="89787312acb34162a6ee3d688b2ed90c" />
 			</Text>
-			<Text BaseName="[string]Text" Height="[float]16" Id="2e6aa89b311c4d18af68745515a29cbb" Left="[float]36" Text="[string]Demonstrates how the Build Array node operates in two modes: concatenate and append." TextWrapping="[TextWrapping]Wrap" Top="[float]194" Width="[float]483" xmlns="http://www.ni.com/PlatformFramework" />
+			<Text BaseName="[string]Text" Height="[float]18" Id="2e6aa89b311c4d18af68745515a29cbb" Left="[float]36" Text="[string]Demonstrates how the Build Array node operates in two modes: concatenate and append." TextWrapping="[TextWrapping]Wrap" Top="[float]194" Width="[float]537" xmlns="http://www.ni.com/PlatformFramework">
```

Comment:
You can add the change in this project to the description i.e. The Font for it was changed to the correct format.